deploy/cdk/lib/manifest-pipeline/deployment-pipeline.ts

```diff
@@ -134,6 +135,23 @@ export class DeploymentPipelineStack extends cdk.Stack {
           cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:*'),
         ],
       }))
+      // all the user to create a backup
+      cdkDeploy.project.addToRolePolicy(new PolicyStatement({
+        actions: [
+          'backup:CreateBackupPlan',
+          'backup:CreateBackupSelection',
+          'backup:CreateBackupVault',
+          'backup:DeleteBackupPlan',
+          'backup:DeleteBackupVault',
+          'backup:DeleteBackupVault',
+        ],
+        resources: [
+          cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:backup-plan:*'),
```

Comment:
Is it possible to further quality the backup-plan and backup-vault - since we're granting DeletePlan and DeleteBackupVault (twice, by the way.  I suspect you may really want a backup:DeleteBackupSelection instead of your second backup:DeleteBackupVault)

Anyway, we don't want this stack to be able to delete backup plans or backup vaults from other stacks.